src/components/skeleton/skeleton.hbs

```diff
@@ -137,4 +137,7 @@
   <div class="bx--select bx--skeleton">
     <select class="bx--select-input"></select>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<p>Data Table</p>
+<table class="bx--skeleton bx--data-table-v2"><thead><tr><th></th><th></th><th></th><th></th><th></th></tr></thead><tbody><tr><td><span></span></td><td><span></span></td><td><span></span></td><td><span></span></td><td><span></span></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></tbody></table>
```

Comment:
Would it help in the future to replace this with formatted HTML? Maybe:

```html
<table class="bx--skeleton bx--data-table-v2">
    <thead>
        <tr>
            <th></th>
            <th></th>
            <th></th>
            <th></th>
            <th></th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td><span></span></td>
            <td><span></span></td>
            <td><span></span></td>
            <td><span></span></td>
            <td><span></span></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
    </tbody>
</table>
```

Reply:
Yea, was just being lazy. I can update. 😄

Reply:
😂